app/views/pages/home.html.erb

```diff
@@ -1 +1,55 @@
-<h1>Welcome to Quizzland!</h1>
+<header>
+  <%= render "./layouts/partials/navbar" %>
+
+  <div class="relative overflow-hidden bg-no-repeat bg-cover" style="
+    background-position: 50%;
+    background-image: url('https://www.nexus-analytics.com.my/wp-content/uploads/2020/10/knowledge-banner.jpg');
+    height: 350px;">
+    <div class="absolute top-0 right-0 bottom-0 left-0 w-full h-full overflow-hidden bg-fixed"
+      style="background-color: rgba(0, 0, 0, 0.35)">
+      <div class="flex justify-center items-center h-full">
+        <div class="text-center text-white px-6 md:px-12">
+          <h1 class="text-5xl font-bold mt-0 mb-6"><%= t("welcome") %></h1>
+          <h3 class="text-3xl font-bold mb-8"><%= t("to_quizland") %></h3>
+          <%= link_to t("first_quiz"), new_quiz_path, class:"inline-block px-6 py-2.5 border-2 border-white text-white font-medium text-xs leading-tight uppercase rounded hover:bg-black hover:bg-opacity-5 focus:outline-none focus:ring-0 transition duration-150 ease-in-out" %>
+        </div>
+      </div>
+    </div>
+  </div>
```

Comment:
I don't thing this block belongs to header. Header mostly is supposed to by displayed on each, and this block is for home page only